src/SokuMod/InLobbyMenu.cpp

```diff
@@ -197,6 +213,10 @@ InLobbyMenu::InLobbyMenu(LobbyMenu *menu, SokuLib::MenuConnect *parent, Connecti
 	strcpy(desc.faceName, "Tahoma");
 	desc.weight = FW_REGULAR;
 
+	// refer to https://stackoverflow.com/a/42333249
+	HMODULE hntdll = GetModuleHandle("ntdll.dll");
+	this->inWine = hntdll && GetProcAddress(hntdll, "wine_get_version");
```

Comment:
I think it would be best to move this and the variable `inWine` to main.cpp. I would probably make that var a char * instead, holding the current wine version (by directly calling the function you get here) and set it to NULL when wine isn't detected.
The globals are defined in data.hpp.